independent-publisher-connectors/Proposal - RegEx Matching (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,466 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "RegEx Matching",
+    "description": "This connector checks whether a text string matches a specified pattern.",
+    "version": "1.0",
+    "contact":{
+      "name": "Mitanshu Garg",
+      "url": "https://twitter.com/mitanshu",
+      "email": "garg.mitanshu@gmail.com"
+    }
+  },
+  "host": "sa018.azurewebsites.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata":[
+    {
+      "propertyName":"Website",
+      "propertyValue":"https://sa018.azurewebsites.net"
+    },
+    {
+      "propertyName":"Privacy Policy",
+      "propertyValue":"https://sa018.azurewebsites.net"
+    },
+    {
+      "propertyName":"Categories",
+      "propertyValue":" Content and Files"

```

Comment:
@happyume please remove the whitespace from the string.

Reply:
This is now updated